recipes-multimedia/omxplayer/omxplayer/0006-Prevent-ffmpeg-configure-compile-race-condition.patch

```diff
@@ -0,0 +1,39 @@
+From 4a13419e3805b541fc58e57f1f27c4a388609ef5 Mon Sep 17 00:00:00 2001
+From: Paul Barker <pbarker@toganlabs.com>
+Date: Thu, 7 Sep 2017 20:02:15 +0000
+Subject: [PATCH] Prevent ffmpeg configure/compile race condition
+
+Additional dependency information is needed in Makefile.ffmpeg to ensure that
+the configure stage is finished before the compile stage starts.
+
+Signed-off-by: Paul Barker <pbarker@toganlabs.com>
+Upstream-status: Inappropriate
```

Comment:
This doesn't sound like an inappropriate patch. Would it make sense to upstream it?

Reply:
I was a bit unsure on what to put as Upstream-status here. We should try to upstream a fix for this and I've opened the bug https://github.com/popcornmix/omxplayer/issues/567 to work towards that. However this patch is based on the previous one and can't be upstreamed exactly as-is.

Reply:
This sounds like Pending, doesn't it?

Reply:
Ok cool I'll change it to Pending.